docs/design/kubevirt-datamover.md

```
@@ -87,18 +94,36 @@ Taking a VolumeSnapshot and then using kopia to process the entire volume and co
     - We need to properly handle cases where we attempt an incremental backup but a full backup is taken instead (checkpoint lost, CSI snapshot restore since last checkpoint, VM restart, etc.)
     - Aborted backups also need to be handled (resulting in a failed PVC backup on the Velero side)
 - DataDownload reconciler (restore)
-  - (this area is less well defined so far, since the kubevirt enhancement doesn't go into as much detail on restore)
-  - We will need a temporary PVC for pulling qcow2 images from object store (if we're restoring the empty temp PVC from backup, that might work here)
-  - We also need PVCs created for each VM disk we're restoring from qcow2 images.
+  - Identify the VM from the DD.
+  - Pull BSL metadata for the VM and backup
+  - Create the temporary PVC to download the qcow2 files onto.
+    - PV here is also temporary
+	- PVC size based on the size of the qcow2 files in BSL needed for restore as well as the PVC sizes
```

markdownlint-cli2 (0.22.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 1

(MD005, list-indent)